docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -121,11 +121,15 @@ ifndef::qs[]
 endif::[]
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click *Create Kafka instance*.
-. Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
-. From the *Cloud region* drop-down menu, select the relevant option. The selection for availability zones is preconfigured as *Single* for a trial instance and *Multi* for a production instance.
+. On the *Kafka Instances* page of the {product-kafka} web console, click *Create Kafka instance*.
+. Complete the *Create a Kafka instance* form to define the following instance details. Some values currently have only one option.
+* *Instance name*: Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
+* *Cloud provider*: Accept the default value of *Amazon Web Services* or select *Google Cloud Platform*.
+* *Cloud region*: Select the relevant region from the list.
```

Comment:
I never loved our use of _relevant_ here. I suggest "Select a cloud provider region from the list".

Reply:
Love what you did here to streamline the steps for creating an instance! :+1:

Reply:
Ditto with my previous comments here.